docs/index.md

```diff
@@ -58,3 +58,9 @@ details for a feature request. [Q&A](https://github.com/NREL/floris/discussions/
 is where to get usage support.
 [Show and tell](https://github.com/NREL/floris/discussions/categories/show-and-tell) is a free-form
 space to show off the things you are doing with FLORIS.
+
+## Timing tests
```

Comment:
This would be more appropriate in the existing code quality page: https://nrel.github.io/floris/code_quality.html

Reply:
Ok sounds good, I'll move it over